pyproject.toml

```diff
@@ -34,6 +34,7 @@ extend-ignore = ["D105", "D107"]  # docstrings for magic methods
 "test/**/*.py" = ["D"]
 [tool.ruff.lint.pydocstyle]
 convention = "numpy"
+property-decorators = ["pytest.fixture"]
```

Comment:
Interesting. Dare I say, relaxes the grammar/style requirement of a fixture's docstring?
- https://docs.astral.sh/ruff/settings/#lint_pydocstyle_property-decorators

Reply:
Yes, that's the idea. Fixtures are frequently nouns rather than verbs, just like properties.